src/main/java/uk/gov/pay/api/exception/ConnectorResponseErrorException.java

```diff
@@ -4,23 +4,19 @@
 
 class ConnectorResponseErrorException extends RuntimeException {
 
-    private int errorStatus;
-    private String errorBody;
+    private int status;
 
     ConnectorResponseErrorException(Response response) {
-        this.errorStatus = response.getStatus();
-        this.errorBody = response.readEntity(String.class);
+        super(response.toString());
```

Comment:
No it is not, is the response displayed in a nice way (better than read Entity, already checked)